Source/ChocolateyGui/Views/RemoteSourceView.xaml

```diff
@@ -220,7 +220,11 @@
                            Target="{Binding ElementName=SearchTextBox}" Content="{x:Static lang:Resources.RemoteSourceView_SearchBoxText}"/>
                     <TextBox x:Name="SearchTextBox" Width="200" FontSize="14"
                              VerticalAlignment="Center"
-                             Text="{Binding SearchQuery, UpdateSourceTrigger=PropertyChanged}" />
+                             Text="{Binding SearchQuery, UpdateSourceTrigger=PropertyChanged}" />
+                    <Button Command="{commands:DataContextCommandAdapter LoadPackages, CanLoadRemotePackages}"
+                            Content="Search"
+                            IsDefault="True"
+                            Style="{DynamicResource IconFlatButtonStyle}" />
```

Comment:
I would suggest to use the MahApps search TextBox style.

```xaml
<TextBox x:Name="SearchTextBox" Width="200" FontSize="14"
            VerticalAlignment="Center"
            mah:TextBoxHelper.ButtonCommand="{commands:DataContextCommandAdapter LoadPackages, CanLoadRemotePackages}"
            Text="{Binding SearchQuery, UpdateSourceTrigger=PropertyChanged}"
            Style="{DynamicResource SearchMetroTextBox}">
    <TextBox.InputBindings>
        <KeyBinding Command="{commands:DataContextCommandAdapter LoadPackages, CanLoadRemotePackages}" Key="Return" />
    </TextBox.InputBindings>
</TextBox>
```
with
```xaml
xmlns:mah="http://metro.mahapps.com/winfx/xaml/controls"
```

![2018-02-05_21h07_02](https://user-images.githubusercontent.com/658431/35826428-0463b59e-0ab9-11e8-9a23-95dfb4b6661c.png)